scripts/pre_push_hook.py

```diff
@@ -167,10 +167,10 @@ def git_diff_name_status(left, right, diff_filter=''):
         diff_filter: arguments given to --diff-filter (ACMRTD...)
 
     Returns:
-        List of FileDiffs (tuple with name/status)
+        list. List of FileDiffs (tuple with name/status)
```

Comment:
Are we not adding a dot at the end of the description in case of both return and args? I see both the cases (ending with dot and not ending with dot).

Reply:
Again -- we should end all descriptions with a dot.

Reply:
Changed the regex to catch dot at the end of description.